### ISSUE TITLE
font-family specification not honored in Docker image

### ISSUE THREAD

**Issue by jonathanlilly** [astefanutti/decktape#79]
Hi, I ran the Docker version of decktape on a large remark.js presentation.  It works great, except that the specified font-family is not used.  I have a serif font, font-family: 'Georgia', specified in the html file, but in the printer slides I get a sans serif font instead, which throws off the slide formatting. Any idea why this is happening?

Thanks!

I'm on a Mac, by the way

**Comment by astefanutti** [comment]
It may be related to #67. I'll try to analyse it ASAP on a minimal use case. In the meantime, a possible work-around is to explicitly define the font instead of just the font-family.

**Comment by jonathanlilly** [comment]
Thanks very much for your quick response.

Here is the online presentation http://www.jmlilly.net/talks/oslo/index.html

And here is a link to the slides

https://www.dropbox.com/s/hq2pyk2b2f53thj/slides.pdf?dl=0

In case that's helpful.

I'd also like to vote in favor of enhancement #49.  Having internal links
reference the pdf document, instead of the web presentation, would be great.

**Comment by astefanutti** [comment]
Thanks for the details. I'll check if `font-family: 'Georgia'` actually resolves correctly in the Docker image. It may be that additional font packages must be installed in the image.

Another alternative for you would be to give the DeckTape Linux binary a try on your system if possible and let me know the result.

> I'd also like to vote in favor of enhancement #49.  Having internal links
reference the pdf document, instead of the web presentation, would be great.

You can add a reaction to the issue, that's kinda similar to a vote 😉.

**Comment by jonathanlilly** [comment]
Thanks for the tip.  I'll give it a try with the binary over the weekend
and let you know the result.  -- J

**Comment by jonathanlilly** [comment]
Using the DeckTape binary did indeed solve the problem.  It looks great now.  Thanks for your help!

**Comment by astefanutti** [comment]
I'm glad it worked! I'll analyse the issue in the Docker image ASAP.

**Comment by astefanutti** [comment]
I've just tried with an updated image, having the `ttf-mscorefonts-installer` package installed in addition, and exporting an HTML page with some text styled with `font-family: 'Georgia'` gives a positif result. In that case, the `Georgia.ttf` font resolves:

    $ fc-match 'Georgia'
    Georgia.ttf: "Georgia" "Normal"

However, with `font-family: serif` it fails, using a sans-serif style font to render the text in the PDF. While in that case:

    $ fc-match serif    
    DejaVuSerif.ttf: "DejaVu Serif" "Book"

So I suspect this is actually similar to #67, where it fails to pick the correct font, as resolved by Fontconfig, whenever a generic font family is used. I'll close this one and follow-up on #67.

Finally, I checked that the DejaVu Serif font installed in the Docker image is correct:

    $ fc-list | grep DejaVuSerif.ttf
    /usr/share/fonts/truetype/dejavu/DejaVuSerif.ttf: DejaVu Serif:style=Book
